ogusa/tests/test_TPI.py

```diff
@@ -166,9 +167,9 @@ def test_twist_doughnut(file_inputs, file_outputs):
     ensure that output returned matches what it has been before.
     '''
     input_tuple = utils.safe_read_pickle(file_inputs)
-    (guesses, r, w, bq, tr, theta, factor, j, s, t, tau_c, etr_params,
+    (guesses, r, w, bq, tr, theta, factor, ubi, j, s, t, tau_c, etr_params,
```

Comment:
@rickecon You don't need to update the `input_tuple` to include the `ubi` object.  Rather, it'll be easier to read in the `input_tuple` as is done and then add a line below to create the `ubi` object.  e.g., `ubi = np.zeros(int(len(guesses) / 2))`.